runtime/vm/jvminit.c

```diff
@@ -4056,6 +4056,18 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 		vm->checkpointState.javaDebugThreadCount = 0;
 	}
 
+#if defined(LINUX)
+	{
+		IDATA enableSupportDisclaimRAMClasses = FIND_AND_CONSUME_VMARG(EXACT_MATCH, VMOPT_XXENABLEDISCLAIMRAMCLASSES, NULL);
+		IDATA disableSupportDisclaimRAMClasses = FIND_AND_CONSUME_VMARG(EXACT_MATCH, VMOPT_XXDISABLEDDISCLAIMRAMCLASSES, NULL);
+		if (enableSupportDisclaimRAMClasses > disableSupportDisclaimRAMClasses) {
+			PORT_ACCESS_FROM_JAVAVM(vm);
```

Comment:
add a check to see if 4k pages are available with `j9vmem_supported_page_sizes()[0]` if not dont set `J9_EXTENDED_RUNTIME2_DISCLAIM_RAM_CLASSES` and output a warning message if the `VMOPT_XXENABLEDISCLAIMRAMCLASSES` is explcitly set.